server/test/footyPoints.test.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require("mongoose")
+const createServer = require("./testingTools")
+var alumniSchema = require('../models/alumniSchema');
+const request = require('supertest');
+
```

Comment:
nit: extra line, I know, we need a linter setup

Reply:
Saw your other comment about this about it being a stylistic choice, and yeah, that's kinda my take.. It's pretty common in Python to do it this way. That said, if in JS that's not the common way to do it, or for this project you want to keep it to one line, I'm happy to do so